[2024-08-07T06:30:10.389+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T06:30:10.428+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Integrate_Time_Location_Dimension.bronze_time manual__2024-08-07T06:29:57.628917+00:00 [queued]>
[2024-08-07T06:30:10.447+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Integrate_Time_Location_Dimension.bronze_time manual__2024-08-07T06:29:57.628917+00:00 [queued]>
[2024-08-07T06:30:10.447+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-07T06:30:10.480+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): bronze_time> on 2024-08-07 06:29:57.628917+00:00
[2024-08-07T06:30:10.484+0000] {standard_task_runner.py:63} INFO - Started process 4310 to run task
[2024-08-07T06:30:10.489+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Integrate_Time_Location_Dimension', 'bronze_time', 'manual__2024-08-07T06:29:57.628917+00:00', '--job-id', '174', '--raw', '--subdir', 'DAGS_FOLDER/etl_time_location_dag.py', '--cfg-path', '/tmp/tmpci72vffa']
[2024-08-07T06:30:10.492+0000] {standard_task_runner.py:91} INFO - Job 174: Subtask bronze_time
[2024-08-07T06:30:10.599+0000] {task_command.py:426} INFO - Running <TaskInstance: Integrate_Time_Location_Dimension.bronze_time manual__2024-08-07T06:29:57.628917+00:00 [running]> on host 45f7764b2bf2
[2024-08-07T06:30:10.748+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Integrate_Time_Location_Dimension' AIRFLOW_CTX_TASK_ID='bronze_time' AIRFLOW_CTX_EXECUTION_DATE='2024-08-07T06:29:57.628917+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-07T06:29:57.628917+00:00'
[2024-08-07T06:30:10.749+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T06:30:10.774+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T06:30:10.775+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 132, in import_optional_dependency
    module = importlib.import_module(name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'xlrd'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/plugins/timelocation/load_date.py", line 15, in readTimeExcel
    time_df = pd.read_excel('./data/SampleDateDim.xls', sheet_name='LoadDates')
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pandas/io/excel/_base.py", line 504, in read_excel
    io = ExcelFile(
         ^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pandas/io/excel/_base.py", line 1580, in __init__
    self._reader = self._engines[engine](
                   ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/pandas/io/excel/_xlrd.py", line 44, in __init__
    import_optional_dependency("xlrd", extra=err_msg)
  File "/home/airflow/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    raise ImportError(msg)
ImportError: Missing optional dependency 'xlrd'. Install xlrd >= 2.0.1 for xls Excel support Use pip or conda to install xlrd.
[2024-08-07T06:30:10.796+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=Integrate_Time_Location_Dimension, task_id=bronze_time, run_id=manual__2024-08-07T06:29:57.628917+00:00, execution_date=20240807T062957, start_date=20240807T063010, end_date=20240807T063010
[2024-08-07T06:30:10.834+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 174 for task bronze_time (Missing optional dependency 'xlrd'. Install xlrd >= 2.0.1 for xls Excel support Use pip or conda to install xlrd.; 4310)
[2024-08-07T06:30:10.863+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-08-07T06:30:10.896+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-07T06:30:10.920+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
